doc/usage.md

```diff
@@ -268,12 +268,10 @@ $fieldQuery->setFieldQuery('title', 'I am a title string');
 $fieldQuery->setFieldParam('title', 'analyzer', 'my_analyzer');
 $boolQuery->addShould($fieldQuery);
 
-$tagsQuery = new \Elastica\Query\Terms();
-$tagsQuery->setTerms('tags', ['tag1', 'tag2']);
```

Comment:
`setTerms()` only accepts a single argument now.
This can also be simplified by passing `field` and `terms` directly in constructor.